operations/patch_util.go

```diff
@@ -221,30 +221,33 @@ func (p *patchParams) validatePatchCommand(ctx context.Context, conf *ClientSett
 		grip.Warningf("warning - failed to set default project: %v\n", err)
 	}
 
-	if err := p.setLocalAliases(conf); err != nil {
-		grip.Warningf("warning - setting local aliases")
-	}
+	// If reusing a previous definition, ignore defaults.
+	if !p.RepeatFailed && !p.RepeatDefinition {
+		if err := p.setLocalAliases(conf); err != nil {
+			grip.Warningf("warning - setting local aliases")
```

Comment:
I know this isn't new, but I'm curious why this is a warning as opposed to informational.

Reply:
Technically users can set the logging level they want for the CLI; theoretically this is a thing they'd expect (but not a thing we'd want to not create patches for) which is why I could imagine warning being the right level (whereas info tends to just be general "submitted /created /etc" logging) but also it could've just been not thought out haha